tiatoolbox/models/dataset/info.py

```diff
@@ -30,14 +30,17 @@
 class DatasetInfoABC(ABC):
     """Define an abstract class for holding dataset information.
 
-    Enforcing such that following attributes must always be defined by the subclass.
+    Enforcing such that following attributes must always be defined by
+    the subclass.
 
     Attributes:
-        inputs (list): A list of paths where each path points to a sample image.
-        labels (list): A list of `int` where each is the label of the sample at
-          the same index.
-        label_names (dict): A dict indicates the possible associate name of each
-          label value.
+        inputs (list):
```

Comment:
Looks like they are defined in KatherPathDataset. Perhaps should be moved / copied here also. @simongraham what do you think?

Reply:
Yes well this `RaiseNotImplemented` won't be triggered because we hard code them as None before the init. We aren't really using these classes at the moment, so it's a bit difficult to know what to do.

Reply:
@shaneahmed  Can I leave this for now and mark as resolved?

Reply:
ok sure